tests/example.rs

```diff
@@ -20,8 +20,8 @@ fn test_artifact_utf8(path: &Utf8Path) -> Result<()> {
 datatest_stable::harness!(
     test_artifact,
     "tests/files",
-    r"^.*/*",
+    r"^.*(?<!\.skip)\.txt",
```

Comment:
Could you make one of them a regular regex? That way we get to see both of the outcomes.